app/controllers/application_controller/filter.rb

```diff
@@ -660,7 +662,8 @@ def exp_commit_find(exp)
                                                           end
       unless ckey.include?("NULL") || ckey.include?("EMPTY")                                # Check for "IS/IS NOT NULL/EMPTY"
         exp[@edit[@expkey][:exp_key]][chk][ckey]["value"] = @edit[@expkey][:exp_cvalue]     #   else set the value
-        exp[@edit[@expkey][:exp_key]][chk][ckey]["value"] += Expression.prefix_by_dot(@edit[@expkey].val2_suffix)
+        new_val = Expression.prefix_by_dot(@edit[@expkey].val2_suffix)
+        exp[@edit[@expkey][:exp_key]][chk][ckey]["value"] += new_val if new_val
```

Comment:
I think a `to_s` would allow us to avoid this double line/variable with an if clause

```suggestion
        exp[@edit[@expkey][:exp_key]][chk][ckey]["value"] += Expression.prefix_by_dot(@edit[@expkey].val2_suffix).to_s
```